uvicorn/supervisors/watchgodreload.py

```diff
@@ -141,12 +148,10 @@ def __init__(
         for w in reload_dirs:
             self.watchers.append(CustomWatcher(w.resolve(), self.config))
 
-    def should_restart(self) -> bool:
+    def should_restart(self) -> Optional[List[Path]]:
```

Comment:
The `self.pause` is not being called here, is it expected?

Reply:
Good catch, fixed.